staging/src/k8s.io/client-go/pkg/api/annotation_key_constants.go

```diff
@@ -19,6 +19,9 @@ limitations under the License.
 package api
 
 const (
+	// PodPresetOptOutAnnotationKey represents the annotation key for a pod to exempt itself from pod preset manipulation
+	PodPresetOptOutAnnotationKey string = "podpreset.admission.kubernetes.io/opt-out"
```

Comment:
Make this `exclude` instead of `opt-out`.  We use `exclude` as a verb in other places in the API, and the requested action is to have podpreset completely ignore this pod.